internal/report/report.go

```diff
@@ -166,13 +212,19 @@ func newPolicyReportResult(policy policiesv1.Policy, resource unstructured.Unstr
 	}
 
 	return &v1alpha2.PolicyReportResult{
-		Source: PolicyReportSource,
-		Policy: name, // either cap-policy_name or ap-policy_name
+		Source:   PolicyReportSource,
+		Policy:   name,     // either cap-policy_name or ap-policy_name
+		Rule:     rule,     // policy name
```

Comment:
I'm not entirely happy with using `policy.GetName()` here (notice that this is different from `policy.GetUniqueName()`, that prepends `cap-` or `ap-` for clusterwide or namespaced policies).

I would prefer to take the name of the Wasm module in the registry. Maybe including even the tag. We could use "net/url" and parse the Url for that part, but that's fragile. The best approach would be to be OCI url aware. Maybe using https://github.com/google/go-containerregistry, but that's a big dep just for this. Worth it doing by hand?

Reply:
Mmhhh, is kyverno using `policies.kyverno.io/title` maybe?

Reply:
I think you are right. I would leave it as it is in the PR.